docs/INSTALL.md

```diff
@@ -96,19 +96,42 @@ docker exec -it <mongo-container-name> bash
 
 # start the mongo command line
 mongo
+```
+```js
+// select and create the admin db
+use admin
+
+// create an admin user with read/write and dbAdmin permissions
+db.createUser({ user: "admin", pwd: "password", roles: [ "readWrite", "dbAdmin" ] } )
 
-# select and create the admin db
+// create the bdd and grid db's
+use bdd
+use grid
+```
+
+#### Using MongoDB version 3
+
+If you are using version 3 for your development environment the authenticaion schema needs to be downgraded. Once you have brought up your Mongo Docker container do the following,
+```bash
+# log into the docker container and open a bash terminal
+docker exec -it <mongo-container-name> bash
+
+# start the mongo command line
+mongo
+```
+```js
+// select and create the admin db
 use admin
 
-# update the auth schema
+// update the auth schema
```

Comment:
Why is this changed?

Reply:
Its due to mongo having different comment identifiers, this allows copy paste of the commands when setting up